app/templates/patch.hbs

```diff
@@ -2,6 +2,7 @@
   <div class="file">
     <div class="meta">
       <span>{{filePath}}</span>
+      <a class="target" {{bind-attr href=targetFileUrl}} title="View generated file at target version">View</a>
```

Comment:
You are correct that the "View" link on GitHub generally links to the new version, but if you look at https://github.com/railsdiff/railsdiff/compare/896cf62...b648959 you'll see for deleted views the link is to the old version.